website/docs/addon_resolve_admin.md

```diff
@@ -15,10 +15,25 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 ## Resolve requirements
-Due to the way resolve handles python and python scripts there are a few steps required steps needed to be done on any machine that will be using AYON with resolve.
+Resolve uses a separate Python interpreter (it does not come with Python itself). The interpreter needs to have installed PySide2 (or PySide6 if on MacOS) and OpenTimelineIO dependencies for Ayon to work correctly. Bellow you can find instructions on how to install them into an installed Python of your choice.
 
-### Installing Resolve's own python 3.6 interpreter.
-Resolve uses a hardcoded method to look for the python executable path. All of tho following paths are defined automatically by Python msi installer. We are using Python 3.6.2.
+Resolve 17 - our recommended version Python 3.6.2
+Resolve 18 - our recommended version range Python 3.9.x
```

Comment:
I've a suggestion, 
```suggestion
- Resolve 17 - our recommended version `Python 3.6.2`
- Resolve 18 - our recommended version range `Python 3.9.x`
```
which enhances the clarity.
![image](https://github.com/ynput/ayon-documentation/assets/20871534/512513f4-85c4-4b17-aa43-813f246090ea)